### PR TITLE
Add Eidolon datamine

### PR DESCRIPTION
**Pull request by nguyentvan7**
## Describe your changes

* Add Eidolon datamine
* Fix some key naming inconsistencies

## Issue or discord link

-

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code, in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] Ran `yarn run mini-ci` locally to validate format + lint.
- [ ] If there were format issues, I ran `nx format write` to resolve them automatically.

### CODE REVIEW

File: libs/sr-formula/src/data/util/read.ts [hunk 1]
```diff
@@ -60,25 +60,25 @@ export class Read extends TypedRead<Tag, Read> {
 
   // Damage Type
   get physical() {
-    return super.with('dt', 'Physical')
+    return super.with('type', 'Physical')
```
Comment (frzyc) [inline libs/sr-formula/src/data/util/read.ts:63]: `type` might be a tad confusing, `dt` is lantua's annoying shorthand, can you call it `dmgType`?

File: libs/sr-stats/src/executors/gen-stats/src/characterData.ts [hunk 2]
```diff
@@ -92,42 +107,63 @@ export default function characterData() {
...
+                ([skillId, levelBoost]) => [
+                  // AttackType fallback to Talent if not defined
+                  DmAttackTypeMap[
+                    avatarSkillConfig[skillId][0].AttackType || 'MazeNormal'
+                  ],
+                  levelBoost,
+                ]
+              )
+            ) as SkillTypeAddLevel,
+            params: rankConfig.Param.map((p) => p.Value),
```
Comment (frzyc) [inline libs/sr-stats/src/executors/gen-stats/src/characterData.ts:156]: probably need to `extrapolateFloat` these numbers?

**Review by frzyc — CHANGES_REQUESTED** [review]

File: libs/sr-stats/Data/Characters/Arlan/data.json [hunk 3]
```diff
@@ -400,5 +400,44 @@
       "crit_dmg_": 0.5000000004656613,
       "taunt": 125
     }
-  ]
+  ],
+  "rankMap": {
+    "1": {
+      "skillTypeAddLevel": {},
+      "params": [
+        0.10000000009313226
```
Comment (frzyc) [inline libs/sr-stats/Data/Characters/Arlan/data.json:408]: values here looks like they should be `extrapolateFloat` to `0.1`

**Review by frzyc — CHANGES_REQUESTED** [review]

File: libs/sr-stats/Data/Characters/Arlan/data.json (hunk 3 above)
Comment (nguyentvan7) [reply libs/sr-stats/Data/Characters/Arlan/data.json:408]: I get a ton of these errors with the new extrapolateFloat calls, and none of these loooong numbers have been fixed
image

only a very small subset are fixed 
image

**Review by frzyc — APPROVED** [review]

**Pull request merged**